### PR TITLE
ESI-12508 Sync ETCD events after post-migration event

### PR DESCRIPTION
**Pull request by jkmar**
After executing command created by getMigrateSubcommandWithPostMigrateEvent post-migration events are synchronized.

### CODE REVIEW

**Comment by marcin-ptaszynski** [comment]
@jkmar , there are no tests for this feature.

Additionally I think this feature should be optional (command option?). I requires etcd to be running during migration. Sometimes ops may decide not to do that - perhaps we need a separate command to push events without server (or just an option in migrate).

File: cli/cli.go [hunk 1]
```diff
@@ -462,6 +462,12 @@ func getMigrateSubcommandWithPostMigrateEvent(subcmd, usage string) cli.Command
 					log.Fatalf("Failed to handle event post-migration, err: %s", err)
 				}
 			}
+
+			syncWriter := server.NewSyncWriter(sync, dbConn)
+			err = syncWriter.Sync()
+			if err != nil {
+				log.Fatalf("Failed to synchronise post-migration events, err: %s", err)
```
Comment (przemyslaw-dobrowolski-cl) [inline cli/cli.go]: synchronize

Comment (jkmar) [inline cli/cli.go]: corrected

**Comment by przemyslaw-dobrowolski-cl** [comment]
@jkmar please rebase

**Pull request merged**